core/README.md

```diff
@@ -0,0 +1,34 @@
+# Core Applications
+
+## TRE API
+
+### Endpoints
+
+#### /ping
+
+- Check if API is alive
+
+### Run locally
+
+Install the requirements on the dev machine
+
+```cmd
+virtualenv venv
+source venv/bin/activate    # linux
+venv/Scripts/activate       # windows
+pip install -r core/api/requirements.txt
+```
+
+Run the webserver locally - from a terminal in the core directory
+
+```cmd
+uvicorn api.main:app
```

Comment:
Caught me out that hadn't changed directory (RTFM). Could a `cd core` or something in the command help?

Reply:
will add that :)